using-valtimo/document/automatic-task-assignment.md

```diff
@@ -0,0 +1,30 @@
+# Automatic task assignment
+
+It is possible to have user tasks be automatically assigned to a Case Assignee making the tasks in a Case appear under 
+the Case Assignee's Task List.
+
+## Functionality
+
+This feature causes User Tasks (existing and new) to be assigned to the current case assignee if there is one assigned.  
+Upon the assignment or change of a user to a case all user tasks from all processes currently using the same business key will 
```

Comment:
I think the first part of this sentence is not that easy to red. Maybe change to '(re-)assignment of a case'